src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum LottoRank {
+    LOSE(0, 0),
+    ONE(6, 2000000000),
+    THREE(5, 1500000),
+    FOUR(4, 50000),
+    FIVE(3, 5000)
+    ;
```

Comment:
당첨 enum 👍

작성해 주신 부분
가독성을 위해서 아래와 같이 표현 하면 더 좋을 것같네요 🙇‍♀️
```suggestion
    LOSE(0, 0),
    ONE(6, 2_000_000_000),
    THREE(5, 1_500_000),
    FOUR(4, 50_000),
    FIVE(3, 5_000)
```

하나더 의견 드리자면 좀 더 의미있는 네이밍이면 어떨까싶습니다 ㅎㅎ
링크 전달드리겠습니다!
https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2002%20-%20%EC%9D%98%EB%AF%B8%20%EC%9E%88%EB%8A%94%20%EC%9D%B4%EB%A6%84.md